devfile.yaml

```
@@ -1,7 +1,7 @@
 schemaVersion: 2.2.0
 metadata:
   name: nodejs
-  version: 1.0.1
+  version: 2.0.0
   displayName: Node.js Runtime
   description: Stack with Node.js 14
```

also update the description here
```suggestion
  description: Stack with Node.js 16
```